Modules/_decimal/_decimal.c

```diff
@@ -5699,9 +5697,29 @@ PyDec_GetConst(const PyObject *v)
     return MPD(v);
 }
 
+static void 
+free_api(void *capi)
+{
+    PyMem_Free(capi);
+}
+
+#define CAPSULE_NAME "_decimal._API"
```

Comment:
`_decimal._API` is used in https://github.com/python/cpython/blob/master/Include/pydecimal.h#L167, I suggest you can move this marco to the pydecimal.h.

Reply:
I agree. I'll rename it to `PyDec_CAPSULE_NAME` for consistence.